src/main/java/pl/tomaszdziurko/jvm_bloggers/blogs/BloggersDataFetchingScheduler.java

```diff
@@ -6,21 +6,34 @@
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Component;
 import pl.tomaszdziurko.jvm_bloggers.TimeConstants;
+import pl.tomaszdziurko.jvm_bloggers.metadata.Metadata;
+import pl.tomaszdziurko.jvm_bloggers.metadata.MetadataKeys;
+import pl.tomaszdziurko.jvm_bloggers.metadata.MetadataRepository;
+import pl.tomaszdziurko.jvm_bloggers.utils.NowProvider;
 
 @Component
 @Slf4j
 public class BloggersDataFetchingScheduler {
 
     private final BloggersDataFetcher bloggersDataFetcher;
+    private final MetadataRepository metadataRepository;
+    private final NowProvider nowProvider;
 
     @Autowired
-    public BloggersDataFetchingScheduler(BloggersDataFetcher bloggersDataFetcher) {
+    public BloggersDataFetchingScheduler(BloggersDataFetcher bloggersDataFetcher,
```

Comment:
You can remove hand-crafted constructor and replace it with `@RequiredArgsConstructor(onConstructor = @__(@Autowired))` class level annotation. See the [example](https://github.com/tdziurko/jvm-bloggers/blob/master/src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/AggregatedRssFeedProducer.java#L32).